functions/EmuScripts/emuDeckCitra.sh

```diff
@@ -55,7 +56,11 @@ Citra_setupSaves(){
 
 #SetupStorage
 Citra_setupStorage(){
-    echo "NYI"
+	local textureLink="$(readlink -f "$Citra_texturesPath")"
+	if [[ "$textureLink" != "$emulationPath/hdpacks/citra" ]]; then
+		rm -rf "$Citra_texturesPath"
+		ln -s "$Citra_texturesPath" "$emulationPath/hdpacks/citra"
+	fi
```

Comment:
this (and the variable on line 7) are the only real changes, the rest is just formatting